tests/test_views.py

```diff
@@ -160,6 +155,7 @@ def test_survey_list_edit(client, user_staff, survey_form_entry,
 
     assert response.status_code == 200
     assert len(surveys) == 1
+    assert surveys[0] == survey_form_entry_observational
```

Comment:
Previously the only contents of `surveys` was `survey_form_entry`, which has `is_active=True` and belongs to the user's Agency but has `published=True`.